### PR TITLE
sync with upstream version 6.1

### PR DESCRIPTION
**Pull request by bjosel**

### CODE REVIEW

File: citation.tsv [hunk 1]
```diff
@@ -73,7 +73,7 @@
 	seriesName	Name	The name of the dataset series		text	69	#VALUE	TRUE	FALSE	FALSE	TRUE	FALSE	FALSE	series	citation	
 	seriesInformation	Information	Can include 1) a history of the series and 2) a summary of features that apply to the series		textbox	70	#VALUE	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	series	citation	
 	software	Software	Information about the software used to generate the Dataset		none	71	,	FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		citation	https://www.w3.org/TR/prov-o/#wasGeneratedBy
-	softwareName	Name	The name of software used to generate the Dataset		text	72	#VALUE	FALSE	TRUE	FALSE	FALSE	FALSE	FALSE	software	citation	
+	softwareName	Name	The name of software used to generate the Dataset		text	72	#VALUE	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	software	citation
```
Comment (aseeland) [inline citation.tsv]: I would not delete the "tab" at the end of the line, since there is another column (termURI).

**Review by aseeland — APPROVED** [review]
Just a small inconsistency... (see line comment).

File: citation.tsv (hunk 1 above)
Comment (bjosel) [reply citation.tsv]: Das hat upstream lustigerweise zw 6.1 und 6.2 entfernt

Comment (aseeland) [reply citation.tsv]: Möglichst wenig Änderungen zu Upstream vs Konsistenz - was gewinnt bei dir?

**Pull request merged**